markdown/ch3.markdown

```diff
@@ -70,11 +70,13 @@ inline` IPython magic to make our images appear below the code:
 
 ```python
 %matplotlib inline
+
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```

Comment:
I think this should be consistent within chapters:
- Either include the comments in all chapters, or only in the first one. (I prefer only the first.)
- Keep the spacing between the lines consistent. (I prefer all together.)
- Make it its own cell, or include it within the first plotting cell. (I prefer its own cell.)

What do you think? My preferences noted above are very mild, but my preference for consistency is strong.